com.valvesoftware.Steam.json

```
@@ -96,6 +96,9 @@
         {
             "name": "lspci",
             "no-autogen": true,
+	    "make-args": [
```

Tab/spaces mix here.
```suggestion
            "make-args": [
```

---

Sure, that's not the reason why this doesn't work though.

---

Just noticed that and commented, sorry.

---

Yeah, it's a good catch. I'm using Emacs and it wants to always put tabs into JSON. Anyhow, completely rewritten, I couldn't manage to get CFLAGS to go through without buildsystem simple (lspci isn't really an autotools project anyway, it just has a Makefile)